zeebe/benchmarks/project/src/main/java/io/camunda/zeebe/config/AppCfg.java

```diff
@@ -18,11 +18,20 @@
 public class AppCfg {
 
   private String brokerUrl;
+  private String operateUrl;
```

Comment:
:thought_balloon: Initially I had it for both worker and starter, but then I changed it to only poll in the starter. So I guess we could push this down if we want... :shrug: